style.css

```diff
@@ -0,0 +1,152 @@
+:root { /* add variables to your css! */
+    --light-blue: rgb(235, 213, 202);
+    --light-pink: rgb(205, 141, 122); 
+    --dark-navy: #5f6a65;
+    --magenta: rgb(173, 162, 177);
+    --white: rgb(255, 255, 255);
+    --light: rgb(242, 241, 241);
+    --dark-blue: #083e25;
+  }
+  
+  html {
+    height: 100%;
+  }
+  
+  body {
+    height: 100%;
+    padding: 0px;
+    margin: 0px;
+    background-color: var(--light-blue);
+  }
+
+  main{
+    padding: 0px 40px 10px;
+  }
+  
+  .main-header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  
+    background-color: var(--dark-navy);
+    color: var(--cg-whitesmoke);
+    padding: 10px 20px;
+  }
+
+  .navbar{
+    margin: 0px 10px;
+    color: var(--white);
+  }
+
+  .header{
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    padding: 10px
+  }
+
+
+  .recipe-heading{
+    background-color: var(--light-pink);
+    padding: 10px 20px;
+    margin: 10px 0px;
+    border-radius: 30px;
+  }
+
+  .recipe-cont
+  {
+    padding: 5px 10px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    
+  }
+
+  .recipe-desc
+  {
+    padding: 5px 10px;
+    flex-basis: 25px;
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+  }
+
+  .recipe-img
+  {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 200px;
+    border: 2px lightgray solid;
+  }
+
+  h2{
+    font-family: cursive;
+    background-color: var(--light-pink);
+    padding: 10px;
+  }
+
+  a{
+    background-color: var(--light-blue);
+    padding:10px;
+   
+  }
+
+  .center{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    width: 50%;
```

Comment:
If you don't want your image to stretch, you can change width: 50%; to max-width: 50% and it should size your image as expected!